calabash-cucumber/lib/calabash-cucumber/launcher.rb

```diff
@@ -221,12 +221,12 @@ def self.instruments?
       # @!visibility private
       def instruments?
         attached_to_gesture_performer? &&
-          gesture_performer.class.send(:name) == :instruments
+          @gesture_performer.class.send(:name) == :instruments
```

Comment:
Was this causing a problem?  I wanted to use the `attr_reader` _gesture_performer_ rather than the instance variable.  It is possible that at some point the accessor might do more than read the instance variable - it might, for example, perform some initialization.